### DISCUSSION TITLE
Copilot changes suggestion in next line

### DISCUSSION THREAD

**Discussion by mdrafiqulrabin** [community/community#6885] [Copilot Conversations] [answered]
Copilot drastically changes the suggestion, just after the line break.

<img width="632" alt="factorial" src="https://user-images.githubusercontent.com/12662633/139202667-66c27b51-c8b5-407f-9b40-c6240462caae.png">

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Hey, thanks for the report. I've added this to our internal issue tracker, and we will investigate what's going on here.
